cookiecutter-django-app/{{cookiecutter.repo_name}}/scripts/update_setup_py.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
Is this file suppose to exist in all new repositories created using cookiecutter-django-app?

Reply:
hm, that's a good point. I'd welcome suggestions on a better place to put it.

Reply:
create a new dir /script in head of this repository and put the script in that